cmd/notation/policy/show.go

```diff
@@ -53,13 +53,16 @@ func runShow(command *cobra.Command, opts showOpts) error {
 	// get policy file path
 	policyPath, err := dir.ConfigFS().SysPath(dir.PathTrustPolicy)
 	if err != nil {
-		return fmt.Errorf("failed to obtain path of trust policy configuration file: %w", err)
+		return fmt.Errorf("failed to obtain path of trust policy file: %w", err)
 	}
 
 	// core process
 	policyJSON, err := os.ReadFile(policyPath)
 	if err != nil {
-		return fmt.Errorf("failed to load trust policy configuration, you may import one via `notation policy import <path-to-policy.json>`: %w", err)
+		if os.IsNotExist(err) {
```

Comment:
nit: New code should use `errors.Is(err, fs.ErrNotExist)`.

See https://pkg.go.dev/os#IsNotExist

Reply:
Thanks! Updated.